core/geometry/src/test/polyface/Polyface.test.ts

```diff
@@ -127,394 +125,299 @@ function exercisePolyface(ck: Checker, polyface: Polyface, longEdgeIsHidden: boo
   }
 }
 
```

Comment:
Below I just moved "verifyFaceData" to line 785 right before the only test that calls it. No more changes. But GutHub diff looks like I made chanegs to "Polyface.HelloWorld" which I did not!